sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/SampleSnippets.cs

```diff
@@ -50,5 +66,72 @@ public async Task BadRequestSnippet()
             }
             #endregion
         }
+
+        [Test]
+        public async Task RecognizeFormContentFromFile()
+        {
+            string endpoint = TestEnvironment.Endpoint;
+            string apiKey = TestEnvironment.ApiKey;
+
+            var credential = new AzureKeyCredential(apiKey);
+            var client = new FormRecognizerClient(new Uri(endpoint), credential);
+
+            string invoiceFilePath = FormRecognizerTestEnvironment.CreatePath("Invoice_1.pdf");
+
+            #region Snippet:FormRecognizerRecognizeFormContentFromFile
+            using (FileStream stream = new FileStream(invoiceFilePath, FileMode.Open))
+            {
+                Response<IReadOnlyList<FormPage>> formPages = await client.StartRecognizeContent(stream).WaitForCompletionAsync();
```

Comment:
I would include a little bit of playing around with the returned model in the snippett. I.e., loop through the tables in this form

Reply:
This is just to add a snippet in the respective .md file showing that we also have the option of using our method with a file stream. We are already showing how to use the data with the Uri method.
to keep our .md files small I opted for this format